lte/gateway/python/magma/smsd/main.py

```diff
@@ -43,5 +46,23 @@ def main():
     service.close()
 
 
+def _get_mme_non_eps_config():
+    """ 
+    Get the NON_EPS_SERVICE_CONFIG setting from the MME. smsd should only be
+    running when this is set to SMS_ORC8R.
+    """
+    mme_service_config = load_service_mconfig("mme", MME())
+    non_eps_service_control = mme_service_config.non_eps_service_control
+    if non_eps_service_control:
+        if non_eps_service_control == 0:
+            return "OFF"
+        elif non_eps_service_control == 1:
+            return "CSFB_SMS"
+        elif non_eps_service_control == 2:
+            return "SMS"
+        elif non_eps_service_control == 3:
+            return "SMS_ORC8R"
+    return "OFF"
+
```

Comment:
this keeps restarting the service. is it expected?
```
vagrant@magma-dev:~/magma/lte/gateway$ sudo service magma@smsd status
● magma@smsd.service - Magma smsd service
   Loaded: loaded (/etc/systemd/system/magma@.service; disabled; vendor preset: enabled)
   Active: activating (auto-restart) since Thu 2021-03-04 16:46:17 UTC; 3s ago
  Process: 7145 ExecStopPost=/usr/bin/env python3 /usr/local/bin/service_util.py smsd (code=exited, status=0/SUCCESS)
  Process: 7137 ExecStart=/usr/bin/env python3 -m magma.smsd.main (code=exited, status=0/SUCCESS)
 Main PID: 7137 (code=exited, status=0/SUCCESS)
vagrant@magma-dev:~/magma/lte/gateway$ sudo service magma@smsd status
● magma@smsd.service - Magma smsd service
   Loaded: loaded (/etc/systemd/system/magma@.service; disabled; vendor preset: enabled)
   Active: activating (auto-restart) since Thu 2021-03-04 16:46:23 UTC; 2s ago
  Process: 7197 ExecStopPost=/usr/bin/env python3 /usr/local/bin/service_util.py smsd (code=exited, status=0/SUCCESS)
  Process: 7189 ExecStart=/usr/bin/env python3 -m magma.smsd.main (code=exited, status=0/SUCCESS)
 Main PID: 7189 (code=exited, status=0/SUCCESS)
vagrant@magma-dev:~/magma/lte/gateway$ sudo service magma@smsd status
● magma@smsd.service - Magma smsd service
   Loaded: loaded (/etc/systemd/system/magma@.service; disabled; vendor preset: enabled)
   Active: activating (auto-restart) since Thu 2021-03-04 16:46:28 UTC; 2s ago
  Process: 7218 ExecStopPost=/usr/bin/env python3 /usr/local/bin/service_util.py smsd (code=exited, status=0/SUCCESS)
  Process: 7209 ExecStart=/usr/bin/env python3 -m magma.smsd.main (code=exited, status=0/SUCCESS)
 Main PID: 7209 (code=exited, status=0/SUCCESS)
```

Reply:
Yes. The service terminates if the MME config is not in the SMS_ORC8R mode and is restarted by systemd. This way, if the MME config is changed from the orc8r, the smsd relay will start in the right mode automatically. If we're concerned about log noise, we can do this config check loop in the service; this approach just seemed cleaner to me.

Reply:
Yes, I think we can go with checking config in the service.
I am bit concerned about ERROR logging.
```
Mar  4 17:49:35 magma-dev magmad[4799]: ERROR:root:GetServiceInfo Error for smsd! [StatusCode.UNAVAILABLE] failed to connect to all addresses
```